### PR TITLE
Show error message if visitor in visitor profile could not be found instead of fatal error

### PR DESCRIPTION
**Pull request by tsteur**
Also added some permissions check. They are not 100% needed as the permissions are checked in the API calls anyway but makes sure they don't get random data from API response and shows always login screen instead of potentially a fatal error.

### CODE REVIEW

File: plugins/Live/Controller.php [hunk 1]
```diff
@@ -114,10 +120,17 @@ public function getVisitorProfilePopup()
     {
         $idSite = Common::getRequestVar('idSite', null, 'int');
 
+        Piwik::checkUserHasViewAccess($idSite);
```
Comment (tsteur) [inline plugins/Live/Controller.php]: Not sure why above we use Common::getRequestVar... instead of `$this->idSite`

Comment (sgiehl) [reply plugins/Live/Controller.php]: Guess it shouldn't make a difference to use `$this->idSite` instead

**Review by sgiehl — COMMENTED** [review]
Should we maybe add UI test(s) for invalid profiles, to prove the behavior doesn't change again?

**Comment by tsteur** [comment]
It is not really for invalid profiles, but more for when the parameter is missing (although would also apply later to possibly deleted profiles once supported by Piwik). Test be useful, need to see when I get to it.

**Pull request merged**